apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -23,12 +24,12 @@ API.v1.addRoute(
 			const config = await cachedSettings({ businessUnit });
 
 			const status = Livechat.online(department);
-			const guest = token && (await Livechat.findGuest(token));
+			const guest: ILivechatVisitor | null = token ? await Livechat.findGuest(token) : null;
 
-			const room = guest && findOpenRoom(token);
+			const room: IOmnichannelRoom | undefined = guest ? findOpenRoom(guest.token) : undefined;
```

Comment:
same here, TS can infer `room` type correctly:
```suggestion
			const room = guest ? findOpenRoom(guest.token) : undefined;
```